### DISCUSSION TITLE
Welcome to DevRev Morocco Discussions!

### DISCUSSION THREAD

**Discussion by larbisahli** [devrev-morocco/devrev-morocco#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our DevRev community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other DevRev  community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sinonodd** [comment]
Hello, First comment^^
My name is kamal, i'm a web developer, i would like to join this community and get to learn something and help out if it needs :smiley:
i ended up here while looking for projects to contribute to :stuck_out_tongue_winking_eye:

**Reply by larbisahli** [reply]
Hey, welcome kamal! nice to have you here, You can join DevRev community on Facebook it's all about tech and startups https://www.facebook.com/devrevmorocco. 
You can ask me any question about the stack in this repo I will be happy to answer them.
